app/views/locations/edit.erb

```diff
@@ -10,6 +10,9 @@
       <%= f.govuk_text_field :address,
                              hint: { text: "For example, 10 High Street, London" } %>
       <%= f.govuk_text_field :postcode %>
+      <%= f.govuk_collection_select :organisation_id, Organisation.all, :id, :name,
```

Comment:
I think the choice should be limited to all orgs the user is a member of, instead of all orgs